API/generic/README.md

```diff
@@ -0,0 +1,48 @@
+Generic API
+===========
+
+#### `id`
+
+> Returns the identity of the Peer
+
+##### `Go` **WIP**
+
+##### `JavaScript` - ipfs.id([callback])
+
+`callback` must follow `function (err, identity) {}` signature, where `err` is an error if the operation was not successful. `identity` is an object with the Peer identity.
+
```

Comment:
Could we add an example to show what "Peer identity" means? Ie. the data that is returned.